draft-ietf-rats-corim.md

```diff
@@ -771,26 +779,24 @@ The types defined for a group identified are UUID and variable-length opaque byt
 
 ##### Measurements
 
-Measurements can be of a variety of things including software, firmware,
-configuration files, read-only memory, fuses, IO ring configuration, partial
-reconfiguration regions, etc. Measurements comprise raw values, digests, or
-status information.
+A measurement can be directly compared with attestation evidence, or it can be a quality that is not directly comparable with evidence.
+The scope of a mroperty is defined by the `environment-map` and optional `mkey` it appears with.
 
-An environment has one or more measurable elements. Each element can have a
-dedicated measurement or multiple elements could be combined into a single
-measurement. Measurements can have class, instance or group scope.  This is
-typically determined by the triple's environment.
+Measurements can be of a variety of things including software, firmware, configuration files, read-only memory, fuses, IO ring configuration, partial reconfiguration regions, etc. Measurements comprise raw values, digests, or status information.
 
-Class measurements apply generally to all the Attesters in the given class.
-Instance measurements apply to a specific Attester instance.  Environments
-identified by a class identifier have measurements that are common to the
-class. Environments identified by an instance identifier have measurements that
-are specific to that instance.
+Each element can have a dedicated measurement/endorsement value or multiple elements could be combined into a single measurement.
```

Comment:
This might benefit from another example.

Reply:
Added. Resolve if it's okay.